handler/area_types.go

```diff
@@ -94,16 +95,30 @@ func (h *AreaTypes) Get(w http.ResponseWriter, r *http.Request) {
 	if res != nil {
 		resp.Count = res.Count
 		resp.TotalCount = res.TotalCount
+
 		for _, edge := range res.Dataset.Variables.Edges {
+
```

Comment:
No need for blank like at beginning of scope

Reply:
Blank line now removed.